Description: Plover.desktop is not a script
 Upstream includes a hashbang at the start of Plover.desktop,
 but we need to remove this, since the file is not a script.
 .
 plover (2.2.0-3) unstable; urgency=low
 .
   * Initial release. Closes: 654659
Author: Thomas Thurman <thomas@thurman.org.uk>
Bug-Debian: http://bugs.debian.org/654659
Last-Update: 2013-06-02

--- plover-2.2.0.orig/application/Plover.desktop
+++ plover-2.2.0/application/Plover.desktop
@@ -1,5 +1,3 @@
-#!/usr/bin/env xdg-open
-
 [Desktop Entry]
 Version=2.2.0
 Type=Application
